src/calendareventmodification.cpp

```diff
@@ -55,129 +55,97 @@ void updateTime(QDateTime *dt, Qt::TimeSpec spec, const QString &timeZone)
 
 }
 
-CalendarEventModification::CalendarEventModification(CalendarData::Event data, QObject *parent)
-    : QObject(parent), m_event(data), m_attendeesSet(false)
+CalendarEventModification::CalendarEventModification(const CalendarStoredEvent *source, QObject *parent)
+    : CalendarEvent(source)
 {
+    if (parent)
+        setParent(parent);
```

Comment:
Why separate setParent() and not as parameter to base class?

Reply:
Good question. Trying to add the `QObject *` argument to the `CalendarEvent` copy constructor, I noticed that I did so because I misinterpreted the compiler error at that time telling that `nullptr` was ambiguous. I thought it was unable to distinguish `Calendar::Event*` to `CalendarEvent*` seeing to `void*`. But no of course. So I changed the `nullptr` by casting `0` to the right type.

It's using proper `QObject*` argument now.